amethyst_utils/src/removal.rs

```diff
@@ -68,18 +58,16 @@ where
 }
 
 /// Removes all entities that have the `Removal<I>` component with the specified removal_id.
-pub fn exec_removal<I, D>(
-    commands: &mut CommandBuffer,
-    subworld: &mut SubWorld<'_>, 
-    removal_id: I,
-) where
+pub fn exec_removal<I, D>(commands: &mut CommandBuffer, subworld: &mut SubWorld<'_>, removal_id: I)
+where
     I: Debug + Clone + PartialEq + Send + Sync + 'static,
```

Comment:
I know you didn't change this, but why are we forcing the use of Debug and Clone here? :thinking: 
(its usage in add_removal is fine however)

Reply:
I guess it's due to trait bounds for `Removal<I>`, which is `Debug + Clone`. So by removing those you won't be able to construct `Removal<I>` for query.